src/recognizers_store.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+package types;
+
+// PatternRecognizer is the object stored in the store
+message PatternRecognizer {
+    // The name of the recognizer, unique
+    string name = 1;
+    // The Regex used by this recognizer
+    string pattern = 2;
+    // The confidence of the result
+    float strength = 3;
```

Comment:
I think we should change it to score - will make more sense, cause this is how we use it.
We can change the API for now, and later change the backend (I can do it)